### DISCUSSION TITLE
TS011F_plug_1 not visible in Energy dashboard in HA

### DISCUSSION THREAD

**Discussion by skibbipl** [Koenkk/zigbee2mqtt#15418] [Devices]
Hi,
I've recently bought Zigbee plug https://www.zigbee2mqtt.io/devices/TS011F_plug_1.html. Luckily it reports correctly attributes. But I cannot see the energy measurements is Energy tab in Home Assistant. Do I miss some configuration or it is not possible to have this information in the Energy tab?

**Comment by TheQue42** [comment]
I am assuming that you have _added_ it to the Energy dashboard? You have to add each device, normally under "Monitor individual devices. I have several of these, and they work fine in the energy dashboard.

**Reply by skibbipl** [reply]
Yes, this is the step I was missing. My other tasmota plug was somehow added automatically to the energy dashboard.
Case solved!
